requirements.txt

```diff
@@ -224,12 +224,12 @@ pyasn1==0.3.7 \
     --hash=sha256:8212bde51ec192e30654efe10e636082738ed728e316049f3685d66b8c92941c \
     --hash=sha256:8d4f0971682203bdfc93740ee7d3fcba0a7f55629451dbe2d32af2335c55b2be \
     --hash=sha256:187f2a66d617683f8e82d5c00033b7c8a0287e1da88a9d577aebec321cad4965
-Jinja2==2.9.6 \
-    --hash=sha256:2231bace0dfd8d2bf1e5d7e41239c06c9e0ded46e70cc1094a0aa64b0afeb054 \
-    --hash=sha256:ddaa01a212cd6d641401cb01b605f4a4d9f37bfc93043d7f760ec70fb99ff9ff
-Werkzeug==0.12.2 \
-    --hash=sha256:e8549c143af3ce6559699a01e26fa4174f4c591dbee0a499f3cd4c3781cdec3d \
-    --hash=sha256:903a7b87b74635244548b30d30db4c8947fe64c5198f58899ddcd3a13c23bb26
+Jinja2==2.10 \
+    --hash=sha256:74c935a1b8bb9a3947c50a54766a969d4846290e1e788ea44c1392163723c3bd \
+    --hash=sha256:f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4
+Werkzeug==0.14.1 \
+    --hash=sha256:d5da73735293558eb1651ee2fddc4d0dedcfa06538b8813a2e20011583c9e49b \
+    --hash=sha256:c3fd7a7d41976d9f44db327260e263132466836cef6f91512889ed60ad26557c
 itsdangerous==0.24 \
```

Comment:
Not part of what you've changed but a _worrying_ package name here?

Reply:
"… so better sign this" - from the docs.

Think it's fine to use internally.

Reply:
Yea it's fine, it's just a name.
On this topic though, we need to have a tidy up of some of the dependency files in SDX, lots of them have changed and requiring the hashes doesn't lend itself to easy upgrading/cleanup.